spec/services/search/location_builder_spec.rb

```diff
@@ -72,27 +91,34 @@
     context "when a non-polygonable location is specified" do
       context "and no radius specified" do
         let(:location) { point_location }
+        let(:radius) { 10 }
+        let(:expected_radius) { 16_093 }
+
+        before { allow_any_instance_of(DistanceHelper).to receive(:convert_miles_to_metres).with(10).and_return(expected_radius) }
```

Comment:
I don't really see the point of doing this?

Reply:
You know what, I don't know. I think it may have been some attempt to not rely on anything else outside of what's being tested, but I assume that's unnecessary for a helper?